master/definitions.xml

```diff
@@ -940,9 +940,10 @@
   <interface name='SVGAnimatedBoolean' href='types.html#InterfaceSVGAnimatedBoolean'/>
   <interface name='SVGUnitTypes' href='types.html#InterfaceSVGUnitTypes'/>
   <interface name='SVGStyleElement' href='styling.html#InterfaceSVGStyleElement'/>
-  <interface name='SVGPoint' href='https://www.w3.org/TR/2014/WD-geometry-1-20140522/#dom-dompoint'/>
-  <interface name='DOMPoint' href='https://www.w3.org/TR/2014/WD-geometry-1-20140522/#dom-dompoint'/>
-  <interface name='DOMPointReadOnly' href='https://www.w3.org/TR/2014/WD-geometry-1-20140522/#dom-dompointreadonly'/>
+  <interface name='SVGPoint' href='https://www.w3.org/TR/2014/CR-geometry-1-20141125/#dom-dompoint'/>
+  <interface name='DOMPoint' href='https://www.w3.org/TR/2014/CR-geometry-1-20141125/#dom-dompoint'/>
+  <interface name='DOMPointInit' href='https://www.w3.org/TR/2014/CR-geometry-1-20141125/#dictdef-dompointinit'/>
```

Comment:
Should we link to the bare https://www.w3.org/TR/geometry-1/ rather than a specific version that will become stale in time? (not sure what the official policy is on this but it always seems weird to me having links to old versions)

Reply:
Since we do it in many places now, I’d assume we should.